### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by Dimoshua**
Request description:
- [DEMO LINK](https://Dimoshua.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Dimoshua.github.io/layout_html-form/report/html_report/)

### CODE REVIEW

**Comment by Dimoshua** [comment]
Request description:
- [DEMO LINK](https://Dimoshua.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Dimoshua.github.io/layout_html-form/report/html_report/)

File: src/index.html [hunk 1]
```diff
@@ -4,14 +4,223 @@
...
+          Yes
+          </label>
+          <label for="nocats">
+            <input type="radio"
+              id  ="nocats"
+              name="Interesting_fact"
+              value="No"
+              required
+            >
+          No
```
Comment (SanyaBratashchuk) [inline src/index.html:121]:
```suggestion
            No
```

File: src/index.html [hunk 2]
```diff
@@ -4,14 +4,223 @@
...
+            id  ="mycolor"
+            name="Interesting_fact"
+            required
+          >
+        </label>
+
+        <label for="mytime" class="block">What time do you go to bed?
+          <input
+            type="time"
+            id  ="mytime"
```
Comment (SanyaBratashchuk) [inline src/index.html:137]:
```suggestion
            id="mytime"
```

File: src/style.css [hunk 3]
```diff
@@ -1 +1,10 @@
 /* styles go here */
+
+.block {
+  margin-bottom: 10px;
+  display: block;
+}
+
+fieldset {
```
Comment (SanyaBratashchuk) [inline src/style.css:8]: use classes for styling purposes

**Review by SanyaBratashchuk — CHANGES_REQUESTED** [review]

**Comment by Dimoshua** [comment]
Request description:
- [DEMO LINK](https://Dimoshua.github.io/layout_html-form/)
- [TEST REPORT LINK](https://Dimoshua.github.io/layout_html-form/report/html_report/)

**Pull request closed without merging**